ansible_deployer/command_line.py

```diff
@@ -20,20 +20,11 @@
 LOG_NAME_FRMT = "ansible-deploy_execution_{}.log"
 
 
-def get_sub_command(command):
```

Comment:
I'm not sure.. this removes the guarantee that subcommand is one of the fixed strings...